Libs/Optimize/ParticleSystem/Sampler.h

```diff
@@ -548,6 +550,7 @@ class Sampler {
   std::vector<std::vector<SphereType>> m_Spheres;
   std::vector<FFCType> m_FFCs;
   std::vector<vtkSmartPointer<vtkPolyData>> m_meshes;
+  bool m_meshFFCMode;
```

Comment:
Recommend default to `false`.  It looks like this would not be an issue for spreadsheet project as it will always call `SetMeshFFCMode`, however the XML input reader will only call this if the XML tag is present.  Otherwise the mode will be undefined and could either be on or off.